shared/css/header.scss

```diff
@@ -107,10 +107,12 @@
         background-color: transparent;
         border: 2px solid $white;
         color: $white;
+        float: right;
```

Comment:
Floating takes this out of the regular flow of the document.  I wonder if a prior element was causing this button to wrap before this was added?